### PR TITLE
Replace links

### PR DESCRIPTION
**Pull request by simondeziel**
There still are some occurrences for `git grep linuxcontainers.org  | grep -vF images.linuxcontainers.org` but those will need more work. We might need to extract content from the discourse instance.

### CODE REVIEW

**Review by monstermunchkin — APPROVED** [review]

**Review by monstermunchkin — APPROVED** [review]
Thanks!

File: doc/metadata.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 site_title: LXD
-site_logo_url: https://linuxcontainers.org/static/img/containers.png
+site_logo_url: https://documentation.ubuntu.com/lxd/en/latest/_static/tag.png
```
Comment (ru-fu) [inline doc/metadata.yaml:2]: Does anybody know what this file is used for?
The doc build doesn't need it.

**Review by ru-fu — APPROVED** [review]

File: doc/metadata.yaml (hunk 1 above)
Comment (simondeziel) [reply doc/metadata.yaml:2]: I'm sorry I have no idea and I couldn't track down what uses it.

Comment (tomponline) [reply doc/metadata.yaml:2]: What does the git history show for this when it was added?

**Review by tomponline — APPROVED** [review]

File: doc/metadata.yaml (hunk 1 above)
Comment (simondeziel) [reply doc/metadata.yaml:2]: It is what provides the link to the logo shown on https://documentation.ubuntu.com/lxd/en/latest/

Comment (ru-fu) [reply doc/metadata.yaml:2]: No, that's in the header: https://github.com/canonical/lxd/blob/dff86455dc9e85de010687aa6224538106da4340/doc/.sphinx/_templates/header.html#L179

I think this file was used to provide the navigation for documentation-builder. Probably we just forgot to remove it.

Comment (simondeziel) [reply doc/metadata.yaml:2]: I've open PR #11981 for that.

**Pull request merged**